packages/driver/src/cypress/command.ts

```diff
@@ -12,7 +14,7 @@ export class $Command {
       // the id prefix needs to be unique per origin, so there are not
       // collisions when commands created in a secondary origin are passed
       // to the primary origin for the command log, etc.
-      attrs.id = _.uniqueId(`cmd-${window.location.origin}-`)
+      attrs.id = `${attrs.chainerId}-cmd-${idCounter++}`
```

Comment:
Discussing chainer and command IDs with @emilyrohrbough in slack, we came to the conclusion that it would be very nice for debugging if it were clear which commands belong to which chainers; thus this change (and the associated one above in `packages/driver/src/cypress/chainer.ts`).

This results in command and chainer IDs that look like this, where it's easy to tell at a glance what belongs to what. It also eliminates the confusion that chainerIds weren't sequential - `_.uniqueId()` shares only a single counter, so you'd get chainer-2, cmd-3, chainer-4, etc.

This is only to make IDs easier to reason about and debug, no user-visible impact.

![image](https://user-images.githubusercontent.com/3003404/192390419-bbd61e34-7e29-475d-a8e5-fd25f415965a.png)